client/src/app/modules/project-page/components/project-page.component.html

```diff
@@ -58,6 +59,7 @@
             <project-quickstarter-list
               [projectQuickstarters]="projectQuickstarters"
               [projectKey]="project.projectKey"
+              [hasPlatformRuntime]="hasPlatformRuntime()"
```

Comment:
why not simply using ``project.platformRuntime`` instead?

Reply:
Yeah, ok, the function was a bit too much maybe ;)